.github/workflows/ci.yml

```diff
@@ -236,103 +195,51 @@ jobs:
         sync-uri = https://raw.githubusercontent.com/257/binpkgs/main
         EOF
 
-    - name: Build ${{ matrix.distro }}/${{ matrix.format }}
-      run: |
-        sudo python3 -m mkosi build
-
-    - name: Boot ${{ matrix.distro }}/${{ matrix.format }}
-      if: matrix.format != 'tar' &&
-          matrix.format != 'cpio'
-      run: sudo ./tests/pexpect/boot.py python3 -m mkosi boot
-
-    - name: Build ${{ matrix.distro }}/${{ matrix.format }} UEFI UKI
-      if: matrix.format != 'directory' &&
-          matrix.format != 'plain_squashfs' &&
-          matrix.format != 'tar' &&
-          matrix.format != 'cpio'
+    - name: Build/Boot ${{ matrix.distro }}/${{ matrix.format }}
+      run: MKOSI_TEST_DEFAULT_VERB=boot sudo python3 -m pytest -m integration tests
+
+    - name: Write shared mkosi config UEFI UKI
```

Comment:
There's no real benefit to having this as a separate step I think. This will never fail so it just adds noise to have it as a separate step. You can merge it with the corresponding build/boot step